src/components/ChecksVisualization/ChecksVisualization.tsx

```diff
@@ -132,13 +132,13 @@ export function ChecksVisualization({ checks, showIcons }: Props) {
         <div>{hoveredCheck?.target}</div>
         <div className={styles.successGaugeContainer}>
           <SuccessRateGauge
```

Comment:
[random thinking for a future pr] we could put the other stats in this tooltip too. Maybe we could let folks choose which stat informs the color of the visualization too?

Reply:
Yes, absolutely 👍